go.mod

```diff
@@ -3,6 +3,8 @@ module get.porter.sh/porter
 go 1.18
 
 replace (
+	// TODO: remove before merging
+	github.com/cnabio/cnab-go v0.23.1 => github.com/vinozzz/cnab-go v0.23.2-0.20220408154326-6a3da71c53b6
```

Comment:
Go ahead and open a PR to cnab-go, as draft, and I can review those changes too

Reply:
https://github.com/cnabio/cnab-go/pull/282/files